Python/getargs.c

```diff
@@ -1979,10 +1980,19 @@ parser_init(struct _PyArg_Parser *parser)
     const char * const *keywords = parser->keywords;
     assert(keywords != NULL);
 
-    if (parser->initialized) {
+    if (_Py_atomic_size_get(&parser->initialized)) {
```

Comment:
Do we even need the atomic operation?  This is effectively a boolean variable (one bit).  The lock you're using takes care of the only race we need to worry about.  So the atomic operation seems unnecessary.

Reply:
Yeah since we are releasing lock after initialized is updated a simple volatile will be sufficient. Updated